### DISCUSSION TITLE
Bash Scripts

### DISCUSSION THREAD

**Discussion by leonelcaroc** [vercel/hyper#7178] [Q&A]
chmod commands is not working into files. what can i do so that file permissions can be changed? i also used `chmod +x main.html` but still not working. 

![chmod](https://github.com/vercel/hyper/assets/81509629/820f014a-1c08-4e35-8a4f-45a0369a1d26)

**Comment by glauciocampos** [comment]
Hi, could you try the following steps:
`sudo lsattr main.html`
`sudo chattr -i main.html`
`sudo chmod 777 main.htlml`
